index.html

```diff
@@ -38,6 +39,156 @@
     </div>
 
   </section>
+
+  <section class="work-section">
+    <div class="RWbox"><div class="RWork"><h3>My Recent works</h3>
+    <div class="workline"></div></div>
+    </div>
+    <div class="work-container">
+      <div class="workone"><img class="workphoto"src="/img/Img Placeholder.svg" alt="Recent work preview">
+       <h5 class="first-Rwork">Multi-Post Stories</h5>
+       <p class="work-dicreption">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+        standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <img class="languages-Fwork" src="/img/Tags.svg" alt="languages used work">
+        <div class="see-box"><button class="see-project" href="index.html">see project</button></div>
```

Comment:
[OPTIONAL] I don't think you need to wrap your buttons in a div. Just style the buttons.